folly/detail/UncaughtExceptionCounter.h

```diff
@@ -27,11 +27,13 @@ struct __cxa_eh_globals;
 // declared in cxxabi.h from libstdc++-v3
 extern "C" __cxa_eh_globals* __cxa_get_globals() noexcept;
 }
-#elif defined(_MSC_VER) && (_MSC_VER >= 1400) // MSVC++ 8.0 or greater
+#elif defined(_MSC_VER) && (_MSC_VER >= 1400) && (_MSC_VER < 1900)// MSVC++ 8.0 or greater
 #define FOLLY_EXCEPTION_COUNT_USE_GETPTD
 // forward declaration (originally defined in mtdll.h from MSVCRT)
 struct _tiddata;
 extern "C" _tiddata* _getptd(); // declared in mtdll.h from MSVCRT
+#elif defined(_MSC_VER) && (_MSC_VER >= 1900) //MSVC++ 2015
```

Comment:
I've had this comment sitting on the Phabricator for a while now, but you probably never saw it:

If this specifically needs Update 1 or greater, you want _MSC_FULL_VER <= 190023506. All you're doing here is checking that it's MSVC 2015.